src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js

```diff
@@ -5,17 +5,21 @@ import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
 const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
   className: 'password-authenticator',
   title () {
-    const daysToExpiry = this.options.appState.get('currentAuthenticator').settings.daysToExpiry;
-    return `${loc('password.expiring.title', 'login', [ daysToExpiry ])}`;
+    const passwordPolicy = this.getPasswordPolicy() || {};
+    const daysToExpiry = passwordPolicy.daysToExpiry;
+
+    if (daysToExpiry > 0) {
```

Comment:
`nit:` This might be a personal preference, I'd rather have two if conditionals with return inside followed by a return outside instead of a if-else ladder.

Reply:
yeah, I usually prefer `if/elseif` unless it getting more complicated. 3 if-else clauses seems ok.